src/Softeq.NetKit.Chat.SignalR/Sockets/ChannelSocketService.cs

```diff
@@ -181,10 +207,10 @@ public async Task DeleteMemberFromChannelAsync(DeleteMemberRequest request)
             var memberToDelete = await _memberService.GetMemberByIdAsync(request.MemberId);
 
             await _channelService.DeleteMemberFromChannelAsync(request.SaasUserId, request.ChannelId, memberToDelete.Id);
-
             await _pushNotificationService.UnsubscribeUserFromTagAsync(memberToDelete.SaasUserId, PushNotificationsTagTemplates.GetChatChannelTag(request.ChannelId.ToString()));
-
             await _channelNotificationService.OnDeletedFromChannel(memberToDelete, request.ChannelId);
+
```

Comment:
Could you please also send system message on Member LeaveChannel. I do understand that this wasn't described in the task. I will extend task description

Reply:
done